frontend/src/ui/Results/Results.tsx

```diff
@@ -105,11 +105,53 @@ const Results: React.FC<ResultsProps> = ({
   >('occupationDisplay');
 
   const chooseToggle = () => {
-    console.log('Toggle!');
-
+    // console.log('Toggle!');
     toggle === 'fill' ? setToggle('opacity') : setToggle('fill');
   };
 
+  const exportChart = '';
```

Comment:
Let's remove the commented out lines and the `exportChart` constant. We can always pull up this this code again using git.